src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -94,6 +94,23 @@ public function storeHasAProductPricedAt($productName, $price = 0)
     }
 
     /**
+     * @Given /^(this product) has "([^"]+)" variant priced at ("[^"]+")$/
+     */
+    public function productHasAVariantPricedAt(ProductInterface $product, $variantName, $price)
```

Comment:
Isn't [it](https://github.com/TheMadeleine/Sylius/blob/product-not-softdeletable-implementation/src/Sylius/Behat/Context/Setup/ProductContext.php#L165) almost the same?w